src/Analysis/AtlasDataDictionary/AtlasDataDictionaryTable/Components/TableRow/TableRowDropdown/TableRowDropdown.tsx

```diff
@@ -47,11 +47,14 @@ const TableRowDropdown = ({
                 className={`expanded-container chart-details-wrapper
                 ${checkIfChartContainsSearchTerm(rowObject, searchTerm)}`}
               >
-                <ValueSummaryChart
-                  chartData={rowObject.valueSummary}
-                  preview={false}
-                  chartType={rowObject.valueStoredAs}
-                />
+                {rowObject.valueSummary
+                  && (
+                    <ValueSummaryChart
+                      chartData={rowObject.valueSummary}
+                      preview={false}
+                      chartType={rowObject.valueStoredAs}
+                    />
+                  )}
```

Comment:
please also add something to show an error icon (❌ ) when `!rowObject.valueSummary`

Reply:
![image](https://github.com/uc-cdis/data-portal/assets/1020732/4f002583-8980-49a3-ae8a-ba63821e2032)
@cemile-ctds thoughts on how to handle this missing data

Reply:
fixed and edited the storybook data to account for the possibility it can be null or empty

Reply:
If there is no data available, we shouldn't show an accordion indicator. If we still want to keep this functionality for all variations, then we should simply display text saying 'No data available' or something similar. What do you think @pieterlukasse?

Reply:
if this is an issue, please open a separate jira

Reply:
I think the error icon is fine, as this is an error situation.